ui/src/containers/Layout.svelte

```diff
@@ -181,11 +358,44 @@
 </body>
 
 <style>
-  .brand {
-    color: white !important;
+  .bg-grey-translucent {
+    background-color: hsla(0, 0%, 40%, 0.1);
+  }
+  .main-container {
```

Comment:
Try using Tailwind classes instead of defining some of the CSS properties here

Reply:
Check global.css

Reply:
Also check that icons don't go onto second line for smaller screen sizes